README.md

```diff
@@ -2,7 +2,191 @@
+It is also one of the most rigorously verified collection of
+formally-verified mathematics in the world; it's verified by
+5 different verifiers written by 5 different people
+in 5 different programming languages.
+
+## What is Metamath?
+
+Metamath is a computer language and associated computer program for
+archiving, verifying, and studying mathematical proofs.
+
+Unlike some other systems, Metamath does not build a particular set
+of axioms into the system. Instead, the Matamath language is simple and robust,
+with an almost total absence of hard-wired syntax.
+In Metamath you express the axioms *and* the theorems in a database.
+To prove a theorem, every proof step *must* be proven using an axiom or
+a previously-proven theorem; as a result, nothing is hidden.
+We believe Metamath provides about the simplest possible framework that
+allows essentially of mathematics to be expressed with absolute rigor.
+
+For more information see
+the [Metamath Home Page](http://us.metamath.org/), the
+[Metamath Proof Explorer Home Page](http://us.metamath.org/mpeuni/mmset.html),
+the [Metamath book](http://us.metamath.org#book), or the
+[video "Metamath Proof Explorer: A Modern Principia Mathematica](https://www.youtube.com/watch?v=8WH4Rd4UKGE).
+
+## What databases are included in this collection?
+
+This is a collection of many databases, but since "set.mm" is by far
+the most actively maintained, you may want to look at it first even if you
+plan to eventually work on something else.
+[The video "Metamath Proof Explorer (set.mm) contributions visualized with Gource through 2019-10-04"]( https://www.youtube.com/watch?v=XC1g8FmFcUU&list=PL1jSu6GGefBm7RBP0Id2Sa9uyVuyhioAC&index=4) shows set.mm's growth over time.
+
+The databases included, in (approximate) decreasing size, are:
+
+* "set.mm" aka Metamath Proof Explorer (MPE) -
+  uses classical logic and
+  Zermelo–Fraenkel set theory with the axiom of choice (ZFC).
+* "iset.mm" aka "Intuitionistic Logic Explorer" -
+  uses intuitionistic logic instead of classical logic,
+  e.g., it does not presume that the law of the implied middle is true
+  in all cases.
+* "nf.mm" aka "New Foundations Explorer" - constructs mathematics using
+  Quine's "New Foundations" (NF) set theory axioms, a direct derivative
+  of the "hierarchy of types" set theory originally presented in
+  Whitehead and Russell's *Principia Mathematica*.
+* "ql.mm" aka "Quantum Logic Explorer" - Starts from the orthomodular
+  lattice properties proved in the Hilbert Space Explorer and takes
+  you into quantum logic.
+* "hol.mm" aka "Higher-Order Logic Explorer" - Starts with HOL (also
+  called simple type theory) and derives equivalents to ZFC axioms,
+  connecting the two approaches.
+* "peano.mm" - Peano arithmetic.
+* "big-unifier.mm" - a unification and substitution test for
+  Metamath verifiers, where small input expressions blow up to thousands
+  of symbols.  It is a translation of William McCune's "big-unifier.in"
```

Comment:
Two spaces after the period is inconsistent with the rest of the document

Reply:
It won't matter since this is markdown that generates HTML. But sure, I'll change it.